samples/tmo_shell/src/tmo_dfu_download.c

```diff
@@ -210,7 +210,7 @@ int tmo_dfu_download(const struct shell *shell, enum dfu_tgts dfu_tgt, char *bas
 			sprintf((char *)dfu_files_rs9116w[0].rfile, "%s.rps",base);
 			dfu_files = dfu_files_rs9116w;
 
-			strncpy(base_url_s,user_base_url_s,sizeof(base_url_s)-1);
+			sprintf(base_url_s,"%srs9116w/",user_base_url_s);
```

Comment:
this line seems like it will cause issues for base URLs other than our binary repo. It also assumes that the base URL ends with a "/".

Maybe it would be better to not append "rs9116w/" to the base URL, and instead instruct users to prepend it to the filename if using our repo. probably need to discuss this with others on the team

Reply:
Yes, have to decide what you want to do there.

Thanks